fineract-loan/src/main/java/org/apache/fineract/portfolio/loanproduct/domain/LoanProduct.java

```diff
@@ -210,6 +210,12 @@ public class LoanProduct extends AbstractPersistableCustom {
     @Column(name = "overdue_days_for_repayment_event")
     private Integer overDueDaysForRepaymentEvent;
 
+    @Column(name = "enable_down_payment", nullable = false)
+    private boolean enableDownPayment;
+
+    @Column(name = "disbursed_amount_percentage_down_payment", scale = 2, precision = 5)
+    private BigDecimal disbursedAmountPercentageDownPayment;
```

Comment:
i am not too fond of this naming. Maybe disbursedAmountPercentage**For**DownPayment?

Reply:
done.